redis/src/test/scala/zio/redis/codecs/ProtobufCodecSupplier.scala

```diff
@@ -0,0 +1,9 @@
+package zio.redis.codecs
+
+import zio.redis.CodecSupplier
+import zio.schema.Schema
+import zio.schema.codec.{BinaryCodec, ProtobufCodec}
+
+object ProtobufCodecSupplier extends CodecSupplier {
+  implicit def codec[A: Schema]: BinaryCodec[A] = ProtobufCodec.protobufCodec
+}
```

Comment:
Perhaps it's worth exposing it via codec supplier's companion constructor, e.g.

```scala
object CodecSupplier {
  def protobuf: CodecSupplier = new CodecSupplier { ... }
}
```

The same applies for string codec.